.travis.yml

```diff
@@ -34,13 +43,15 @@ before_script:
     make install &&
     popd &&
     popd
-  - travis_retry wget https://github.com/scafacos/scafacos/releases/download/v1.0.1/scafacos-1.0.1.tar.gz &&
-    tar -xzf scafacos-*.tar.gz &&
-    pushd scafacos* &&
-    ./configure --enable-fcs-solvers=direct,p2nfft,p3m,ewald,fmm --prefix=$HOME/scafacos CC=mpicc CXX=mpicxx FC=mpif90 &&
-    make -j2 &&
-    make install &&
-    popd
+  - if [[ ${SCAFACOS} ]]; then
```

Comment:
Same here.

To be clear I have nothing against the changes but I think you should either capture that in the description, or (better) do that in a separate PR.

Reply:
Done!